package.json

```diff
@@ -84,7 +86,7 @@
     "release-it": "^14.11.6",
     "release-it-lerna-changelog": "^3.1.0",
     "rimraf": "^3.0.2",
-    "typescript": "^4.4.4",
+    "typescript": "^5.0.4",
```

Comment:
looks like this didn't change -- I didn't even check if this project already had TS available 😅